e2e-test/README.md

```diff
@@ -2,7 +2,7 @@
 
 This folder contains end-to-end tests which test the whole functionality of the application.
 
-For the setup of the e2e-tests please visit the [Developer-Setup](https://github.com/openkfw/TruBudget/blob/master/doc/installation/Developer-Setup.md#tests)
+For the setup of the e2e-tests please visit the [Developer Setup](https://github.com/openkfw/TruBudget/blob/master/docs/developer/developer-setup.md#tests)
```

Comment:
**DO NOT** use absolut path to GitHub. Use relativ link. This will break the documentation!

Reply:
done - There are a lot of links, this should be resolved in a new issue

Reply:
no. most of the direct links point to a readme in a sub project this is fine. this link points to the doc folder. pls use relative link instead.